tests/test_math_stat.py

```diff
@@ -0,0 +1,96 @@
+import aesara.tensor as at
+import numpy as np
+import pytest
+from aesara.tensor.random.basic import NormalRV
+
+from aeppl.rewriting import construct_ir_fgraph
+
+
+@pytest.mark.parametrize(
+    "mu_x, mu_y,  x_shape, y_shape, new_rv_mu",
+    [
+        (
+            np.array([1, 10, 100]),
+            np.array(2),
+            None,
+            None,
+            np.array([3, 12, 102]),
+        ),
+        (np.array([1, 10, 100]), np.array(2), (), (), np.array([3, 12, 102])),
+        (
+            np.array([1, 10, 100]),
+            np.array(2),
+            (5, 3),
+            (),
+            np.broadcast_to(np.array([3, 12, 102]), (5, 3)),
+        ),
+        (
+            np.array([[1, 10, 100]]),
+            np.array([[0.2], [2], [20], [200], [2000]]),
+            None,
+            None,
+            np.array(
+                [
+                    [1.2, 10.2, 100.2],
+                    [3, 12, 102],
+                    [21, 30, 120],
+                    [201, 210, 300],
+                    [2001, 2010, 2100],
+                ]
+            ),
+        ),
+        (
+            np.broadcast_to(np.array([1, 10, 100]), (5, 3)),
+            np.array([2, 20, 200]),
+            (2, 5, 3),
+            None,
+            np.broadcast_to(np.array([3, 30, 300]), (2, 5, 3)),
+        ),
+        (
+            np.array([[1, 10, 100]]),
+            np.array([[0.2], [2], [20], [200], [2000]]),
+            (2, 5, 3),
+            None,
+            np.broadcast_to(
+                np.array(
+                    [
+                        [1.2, 10.2, 100.2],
+                        [3, 12, 102],
+                        [21, 30, 120],
+                        [201, 210, 300],
+                        [2001, 2010, 2100],
+                    ]
+                ),
+                (2, 5, 3),
+            ),
+        ),
+        (
+            np.array(1),
+            np.array(6),
+            (5, 1),
+            (1,),
+            np.full((5, 3), 7),
+        ),
+    ],
+)
+def test_add_independent_normals(mu_x, mu_y, x_shape, y_shape, new_rv_mu):
+    srng = at.random.RandomStream(29833)
+
+    X_rv = srng.normal(mu_x, 0.03)
+    X_rv.name = "X"
+
+    Y_rv = srng.normal(mu_y, 0.04)
+    Y_rv.name = "Y"
+
+    Z_rv = X_rv + Y_rv
+    Z_rv.name = "Z"
+    z_vv = Z_rv.clone()
+
+    fgraph, _, _ = construct_ir_fgraph({Z_rv: z_vv})
+
+    new_rv = fgraph.outputs[0].owner.inputs[0]
+
+    assert isinstance(new_rv.owner.op, NormalRV)
+    assert np.allclose(new_rv.owner.inputs[3].eval(), new_rv_mu)
+    assert np.allclose(new_rv.owner.inputs[4].eval(), 0.05)
```

Comment:
Maybe we should make sure broadcasting also works when `sigma` is not a scalar?

Reply:
When I was writing these tests, I thought that testing it on one parameter was sufficient under the assumption that `srng.normal` handles broadcasting the `sigma` the same way it does for `mu`. Is this the case? Would it still be better to include `sigma` broadcasting tests?